### PR TITLE
My Coordinants

### PR DESCRIPTION
**Pull request by ramprasadkolla4**
@githubteacher

### CODE REVIEW

**Comment by renjian2017** [comment]
@ramprasadkolla4 This looks great! 🎉

**Comment by irezumi10111** [comment]
New Zealand is a great place to add to the map!

**Comment by shajanjp** [comment]
@ramprasadkolla4 change the `githubHandle` value to your github username.

**Comment by terickso25** [comment]
@

**Comment by terickso25** [comment]
@ramprasadkolla4 github is great!

File: _pins/coordinants.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: tester
```
Comment (hectorsector) [inline _pins/coordinants.json]: @ramprasadkolla4 you'll want to ensure that this is your actual GitHub Handle, and not `tester`

**Comment by hollenberry** [comment]
Hey @ramprasadkolla4  ! You're so close to finishing this PR! As noted by Hector above, if you make changes to the file name and GitHub handle, you'll be set. 👍

Please let us know if you need any help! We'll be closing this out to keep the repository clean if there's no action or conversation within the week.

**Comment by crichID** [comment]
@ramprasadkolla4 I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/4674/commits/3e49d893e23ef8abe3f07f9161894c80d12a64c5. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by April 21st, I will go ahead and do it just to keep the repository clean!

**Pull request merged**